code/index.html

```diff
@@ -17,21 +18,20 @@
         <img id="killerImage" src="" height="100" />
         <p id="killerDescription"></p>
```

Comment:
If you in the future would like to style the killerDescription separately it could be a good idea to add a class 😄